### PR TITLE
Added product listing

### LINKED ISSUE #1: Add product listing

### PR DESCRIPTION
**Pull request by ghost**
Closes #1

### CODE REVIEW

File: src/Lwi/Client/ServiceDescription/Inside.php [hunk 1]
```diff
@@ -77,6 +77,32 @@
...
+            'uri'              => 'products',
+            'summary'          => 'List products',
+            'parameters'       => array(
+                'page' => array(
+                    '$ref' => 'PageParam',
+                ),
+                'page_size' => array(
+                    '$ref' => 'PageSizeParam',
+                ),
+                'query' => array(
```
Comment (ivan-wolf) [inline src/Lwi/Client/ServiceDescription/Inside.php]: Is this param really implemented for products? As far as I know, the param is used only for listing assets.

File: examples/README.md [hunk 2]
```diff
@@ -6,3 +6,4 @@ Currently, the following examples can be found here:
 | [asset-info.php](asset-info.php)           | Fetch a single asset                                           |
 | [asset-list.php](asset-list.php)           | List assets (paging, filtering and sorting are supported)      |
 | [asset-type-list.php](asset-type-list.php) | List asset-types (paging, filtering and sorting are supported) |
+| [product-list.php](product-list.php)       | List products (paging and sorting are supported)               |
```
Comment (ivan-wolf) [inline examples/README.md]: Why is filtering not mentioned? It is supported by the API.

File: src/Lwi/Client/ServiceDescription/Inside.php (hunk 1 above)
Comment (ghost) [inline src/Lwi/Client/ServiceDescription/Inside.php]: That's right, my mistake.

File: examples/README.md (hunk 2 above)
Comment (ghost) [inline examples/README.md]: Because I didn't implement it, I thought that it is not necessary for the client implementation, should I?

Comment (ivan-wolf) [inline examples/README.md]: I'm not sure if it's needed or not for the homepage. Please add it.

Comment (ghost) [inline examples/README.md]: ok

**Pull request merged**